app/scripts/metamask-controller.js

```diff
@@ -2192,9 +2189,17 @@ export default class MetamaskController extends EventEmitter {
     this.extension.runtime.onMessageExternal.addListener(onMessageReceived);
     // Fire a ping message to check if other extensions are running
     checkForMultipleVersionsRunning();
+
+    if (this.onboardingController.store.getState().completedOnboarding) {
+      this.networkController.lookupNetwork();
+    }
   }
 
-  triggerNetworkrequests() {
+  async triggerNetworkrequests() {
```

Comment:
Is this `async` needed?

Reply:
reverted this [on this commit](https://github.com/MetaMask/metamask-extension/pull/23622/commits/1b78a964ec2093f65d91ce1e3d79810ae8f4eff2)